code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -178,6 +178,7 @@
 /obj/item/light_eater/Initialize(mapload)
 	. = ..()
 	ADD_TRAIT(src, TRAIT_NODROP, HAND_REPLACEMENT_TRAIT)
+	ADD_TRAIT(src, TRAIT_DOOR_PRYER, HAND_REPLACEMENT_TRAIT)
```

Comment:
Hand replacement trait? Should be innate

Reply:
👍